frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -244,13 +244,13 @@ export function Experiment(): JSX.Element {
                                                 numbers, hyphens, and underscores.
                                             </div>
                                             <div className="variants">
-                                                {experiment.parameters.feature_flag_variants?.map((variant, index) => (
+                                                {experiment.parameters.feature_flag_variants?.map((_, index) => (
                                                     <Group
                                                         key={index}
                                                         name={['parameters', 'feature_flag_variants', index]}
                                                     >
                                                         <div
-                                                            key={`${variant.key}-${index}`}
+                                                            key={`variant-${index}`}
```

Comment:
Yeah, this should fix it.
Though I don't see why there's a key here at all. Normally there shuold be a key on the outermost mapped element, but its children don't need one for anything. So here, `<Group>` should be keyed, but not this inner div. Unless I'm missing something tricky here?